srcpkgs/EncryptPad/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'encryptpad'
+pkgname=EncryptPad
+version=0.5.0.1
+revision=1
+build_style=configure
+configure_script="./configure.py"
+hostmakedepends="pkg-config python3"
+makedepends="botan-devel bzip2-devel qt5-devel zlib-devel"
+short_desc="Minimalist secure text editor and file encryptor"
+maintainer="Zyxer Zyxani <zyxerzyxani@cryptolab.net>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/evpo/EncryptPad"
+changelog="https://github.com/evpo/EncryptPad/blob/master/CHANGES.md"
+distfiles="https://github.com/evpo/EncryptPad/archive/v${version}.tar.gz"
+checksum=8518864441d25df58e12d9d400d256a7ae1ff20dc9037954a0c8298d4d1211f5
+
+do_install() {
```

Comment:
does this not have a `make install` target?

Reply:
Yes it has? 
"vbin bin/release/encryptcli
	vbin bin/release/encryptpad

	vinstall encryptpad.desktop 644 usr/share/applications
	vinstall encryptpad.xml 644 usr/share/mime/packages

	for res in 16x16 32x32 128x128; do
		vinstall images/icns.iconset/icon_${res}.png 644 usr/share/icons/hicolor/${res}/apps encryptpad.png
	done
}"
![image](https://user-images.githubusercontent.com/33463214/180659355-eadf7912-9f30-4b2f-8d3d-6669efa72a03.png)

Reply:
no I meant, does the program's build system generate a makefile that has an `install` target, so you don't have to do it manually. If so, that can be used instead of manually installing everything, and things that are not included in that can be installed in `post_install()`

Reply:
The EncryptPad make install points to scripts/install.sh which ignores PREFIX and installs files as root unconditionally. It makes more sense to just do the build step and then install relevant files manually.

Reply:
ok, carry on then